noxfiles/constants_nox.py

```diff
@@ -28,7 +28,7 @@ def get_current_tag() -> str:
 # Disable TTY to perserve output within Github Actions logs
 # CI env variable is always set to true in Github Actions
 # The else statement is required due to the way commmands are structured and is arbitrary.
-CI_ARGS = "--no-TTY" if getenv("CI") else "--user=root"
+CI_ARGS = "-T" if getenv("CI") else "--user=root"
```

Comment:
Two things here:
1. This change seems unrelated to the larger changeset on this PR. I would prefer if a separate issue/PR were opened to properly document the context here.
2. Looking at [the `docker compose run` usage docs](https://docs.docker.com/compose/reference/run/), it looks like `--no-TTY` is no longer supported and `-T` is the appropriate choice, but this change appears to have caused CI failures.

Reply:
Ah yeah, I had been testing out a change Steve suggested in this PR: https://github.com/ethyca/fides/pull/558 to confirm that tests that should fail, do fail. (The switch to `.removesuffix` didn't fail before this, but now it does).

I'll revert this commit and wait for Steve's PR to go in, then merge, if that sounds 👍

Reply:
Sounds 👍 😄